Sources/AngouriMath/Core/Entity/Continuous/Entity.Continuous.Complex.Definition.cs

```diff
@@ -36,6 +37,12 @@ namespace AngouriMath
                 /// Imaginary part of a complex number
                 /// </summary>
                 public Real ImaginaryPart => imaginary ?? Integer.Zero;
+                
+                /// <summary>
+                /// Conjugate of a complex number. Given this = a + ib, Conjugate = a - ib
+                /// </summary>
+                public Complex Conjugate => conjugate.GetValue(@this => Create(RealPart, -ImaginaryPart), this);
```

Comment:
I'm not sure it's a static lambda. Can you add `static` before `@this` to see? Like, `static @this =>`. I suspect there might be a capture of RealPart field there.

Reply:
I used the other classes as an example, but do you mean something like:
```cs
public Complex Conjugate => conjugate.GetValue(static @this => Create(@this.RealPart, -@this.ImaginaryPart), this);
```

Reply:
Yeah, that's what I meant. Otherwise it would be very costly to call Conjugate every time, you know

Reply:
I got it. Is there any reason to not use that on the `Rational` cached properties?
```csharp
public Integer Denominator => denominator.GetValue(@this => ERational.Denominator, this);
private FieldCache<Integer> denominator;
```

Reply:
Nope, there's no reason, I think you just found a bug :). Could you fix it too?

Reply:
Sure, I'll do it in another PR. Thanks for all the suggestions.

Reply:
I think we should do some benchmarking to see if that gives any real performance gains since, according to the [Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/static-anonymous-functions), its purpose is to avoid unintentionally accessing the instance context.

Reply:
Yes, because if you access, you unintentionally create a capturing lambda. You can check sharblab.io on lambdas *with* capture and without. For the former, you will get, that it always creates a new instance of a reference type (while the latter is just something like method).

Benchmark would be useful, but probably in a repo related to `FieldCache`, [here](https://github.com/WhiteBlackGoose/FieldCache). FieldCache is by far faster than conditional weak table (without allocations), so I think there will be a huge difference for with vs without capture lambdas. If you want, you can PR addition of record similar to [this one](https://github.com/WhiteBlackGoose/FieldCache/blob/main/FieldCache/Benchmark/Program.cs#L37) but with capture (`@this.a` to `a`).

Reply:
Thanks for the information, I'll take a look after my classes.

Reply:
@jaimeadf hey... are you going to PR [that](https://github.com/asc-community/AngouriMath/pull/420#discussion_r635385506) fix? 😃

Reply:
Okay. I'll do it myself